website/demos/components/DraggableHeaderRenderer.tsx

```diff
@@ -11,31 +11,52 @@ export function DraggableHeaderRenderer<R>({
   column,
   ...props
 }: DraggableHeaderRendererProps<R>) {
-  const [{ isDragging }, drag] = useDrag({
-    type: 'COLUMN_DRAG',
-    item: { key: column.key },
-    collect: (monitor) => ({
-      isDragging: monitor.isDragging()
-    })
-  });
-
-  const [{ isOver }, drop] = useDrop({
-    accept: 'COLUMN_DRAG',
-    drop({ key }: { key: string }) {
-      onColumnsReorder(key, column.key);
-    },
-    collect: (monitor) => ({
-      isOver: monitor.isOver(),
-      canDrop: monitor.canDrop()
-    })
-  });
+  const [isDragging, toggleDragging] = useState(false);
+  const [isOver, toggleIsOver] = useState(false);
```

Comment:
I don't mind too much, but why `toggle` instead of `set`? It's not consistent with pretty much everything else